http_calls.py

```diff
@@ -91,6 +100,7 @@ class RestTools(HttpCall):
     http_headers = {'Content-Type': 'application/json', 'Accept': 'application/json'}
 
     def getRequest(self, url, data=None, headers={}):
+        headers.update(g_headers)
```

Comment:
Actually i don't see much sense in `local headers` at all. Didn't delete them, just because was afraid to break something in existing AT suites. Again we are talking about Acceptance Tests here - should be visible and transparent what's going on. And imho the usual pattern for ATs is headers are re-used almost always.